### DISCUSSION TITLE
Publicação simplificada - Material de Estudos

### DISCUSSION THREAD

**Discussion by jcaique** [HashLoad/horse#369] [Show and tell]
Olá gente, estou iniciando os estudos do framework e gostaria de fazer algumas publicações para testes apenas. 
Neste quesito, gostaria de saber se vocês possuem algum material de estudos que possa ser disponibilizado,
algo como um passo a passo e quais a possíveis formas de fazer a publicação de uma API Horse. Não encontrei nada aprofundado. 

Grato.

**Comment by viniciussanchez** [comment]
Bom dia... tudo o que temos são exemplos no github, vídeos, artigos, palestras... e grupos de ajuda
